.github/environments/from_13/config.yml

```diff
@@ -0,0 +1,7 @@
+K8S_NAMESPACE: testing-openedx
+PLUGINS:
+- forum
+- mfe
+- drydock
+DRYDOCK_INIT_JOBS: true
+DRYDOCK_MIGRATE_FROM: 13
```

Comment:
I'm not sure if we need to test DRYDOCK_MIGRATE_FROM for every release we migrate from. Could we just test it on the "enabled" case by setting DRYDOCK_MIGRATE_FROM to the oldest release we support migration from?

Reply:
There are some migration jobs that are only enabled when the version is specific, like the jobs that apply only on the 14 release.

Reply:
@Ian2012 The inclusion of the migration job is controlled by the `DRYDOCK_MIGRATE_FROM` variable, as you can see [here](DRYDOCK_MIGRATE_FROM), if the variable is set to a truthy value, the is added. Then, inside the job in the `args` parameter, and based on the `DRYDOCK_MIGRATE_FROM` value, we adjust the command. Kubeconform won't check if the command is valid, so I think we don't need to test every value of `DRYDOCK_MIGRATE_FROM`.